tests/gnrc_dhcpv6_client/tests-as-root/01-run.py

```diff
@@ -57,24 +57,35 @@ def extract_global_addresses(child):
 
 def check_ia_na_addr(ia_na_addr):
     """Check if the expected IA_NA address has been assigned"""
-    return IPv6Address(ia_na_addr) in IPv6Network("{}/64".format(IA_NA_ADDRESS_POOL_PREFIX))
+    res = IPv6Address(ia_na_addr) in IPv6Network("{}/64".format(IA_NA_ADDRESS_POOL_PREFIX))
+    print("check_ia_na_addr", ia_na_addr, res)
```

Comment:
Please remove the `print`s from the tests. These were only meant for me to understand the results of the tests and for you to understand my provided output ;-).

Reply:
Oh, I see, sorry ;)